kirstie/resume.md

```diff
@@ -18,6 +18,7 @@ As part of the 2016 Mozilla Fellows for Science program, I completed/participate
 
```

Comment:
Feel free to link to your where file in the front-matter at the top, as well as the other files that we talked about this morning. I believe you integrated aspects of them below but they're great resources for anyone who ends up on your resume page and wants to dive deeper.